xml/System/TimeOnly.xml

```diff
@@ -1599,11 +1668,12 @@
         <Parameter Name="result" Type="System.TimeOnly" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="s">To be added.</param>
-        <param name="formats">To be added.</param>
-        <param name="result">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="s">The string containing time to parse.</param>
+        <param name="formats">An array of allowable formats of s.</param>
+        <param name="result">When this method returns, contains the timeOnly value equivalent to the time contained in s, if the conversion succeeded, or MinValue if the conversion failed. The conversion fails if the s parameter is Empty, or does not contain a valid string representation of a time. This parameter is passed uninitialized.</param>
+        <summary>Converts the specified string of a time to its TimeOnly equivalent and returns a value that indicates whether the conversion succeeded.</summary>
+        <returns>
+          <see langword="true" /> if the s parameter was converted successfully; otherwise, <see langword="false" />.</returns>
```

Comment:
```suggestion
        <param name="s">The string containing the time to parse.</param>
        <param name="formats">An array of allowable formats of <paramref name="s" />.</param>
        <param name="result">When this method returns, contains the <see cref="T:System.TimeOnly" /> value equivalent to the time contained in <paramref name="s" />, if the conversion succeeded, or <see cref="F:System.TimeOnly.MinValue" /> if the conversion failed. The conversion fails if <paramref name="s" /> is an empty string, or does not contain a valid string representation of a time. This parameter is passed uninitialized.</param>
        <summary>Converts the specified string of a time to its <see cref="T:System.TimeOnly" /> equivalent and returns a value that indicates whether the conversion succeeded.</summary>
        <returns>
          <see langword="true" /> if <paramref name="s" /> was converted successfully; otherwise, <see langword="false" />.</returns>
```